lib/Text/BibTeX.pm

```diff
@@ -26,7 +26,9 @@ require DynaLoader;
 
 our $VERSION='0.74';
 
-@ISA = qw(Exporter DynaLoader);
+#@ISA = qw(Exporter DynaLoader);
+use base qw(Exporter DynaLoader);
```

Comment:
In fact, it seems that parent is not available in Perl 5.8 by default, so I would like to stick to @ISA.
Sorry